pwncat/__main__.py

```diff
@@ -204,11 +232,14 @@ def main():
                 # Locate all installed implants
                 for target in db.root.targets:
 
-                    if target.guid != host and target.public_address[0] != host:
+                    if (
+                        target.guid != query_args["host"]
+                        and target.public_address[0] != query_args["host"]
+                    ):
                         continue
 
                     # Collect users
-                    users = {}
+                    userss = {}
```

Comment:
@calebstewart Is this a _typo_? If there are no installed `implant`s, this _bug_ will not arise.

Reply:
Yes, it is. There was the same typo in the connect command as well. I've fixed them both.